File: R/mysql-connect.r
install.packages("RMySQL")
library(RMySQL)

con <- dbConnect(MySQL(), user = 'yourusername', password = 'yourpassword', host = 'localhost', dbname = 'yourdatabase')

result <- dbSendQuery(con, "SELECT * FROM yourtable")
data <- fetch(result)

print(data)

dbDisconnect(con)

File: Swift/mysql-connect.swift
import MySQL

let connection = try MySQL.Connection(
    host: "localhost",
    user: "yourusername",
    password: "yourpassword",
    database: "yourdatabase"
)
try connection.open()

let statement = try connection.prepare(query: "SELECT * FROM yourtable")
let results = try statement.execute()

for row in results {
    print(row)
}

connection.close()